python/paddle/fluid/contrib/slim/tests/QAT_mkldnn_int8_readme.md

```diff
@@ -122,7 +120,7 @@ cd /PATH/TO/DOWNLOAD/MODEL/
 # export MODEL_NAME=resnet50
 # export MODEL_FILE_NAME=QAT2_models/${MODEL_NAME}_quant.tar.gz
 wget http://paddle-inference-dist.bj.bcebos.com/int8/${MODEL_FILE_NAME}
-mkdir ${MODEL_NAME} && tar -xvf ResNet50_qat_model.tar.gz -C ${MODEL_NAME}
+mkdir ${MODEL_NAME} && tar -xvf ${MODEL_NAME}_qat_model.tar.gz -C ${MODEL_NAME}
```

Comment:
Here, the `ResNet50_qat_model.tar.gz` is case sensitive with capital letters.

Reply:
If we use ResNet50, we have to change ${MODEL_NAME} before it too. 
Another thing is this extraction is only for QAT1.0 models, because all QAT1.0 models after extraction are called `models/` folder.
Maybe let's just delete this line, users may figure it out

Reply:
I meant you can keep `ResNet50_qat_model.tar.gz` instead of `${MODEL_NAME}_qat_model.tar.gz`.

Reply:
Ok, I think splitting this code section into two sections for QAT1 and QAT2 would make it more clear.